docs/reference/commandline/cp.md

```diff
@@ -89,6 +89,25 @@ you must be explicit with a relative or absolute path, for example:
 
     `/path/to/file:name.txt` or `./file:name.txt`
 
+## Examples
+
+Copy a local file into container
+```console
```

Comment:
While you're fixing the H2 -> H3, could you also put a blank line between the text and the "code" block? It's not strictly needed (we did have issues with that in the past, but I think those are solved), but it's consistent with how we use them everywhere

So;

    Some text before the example code
    
    ```console
    $ echo "I am the example code"
    ```